src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -165,16 +163,8 @@ export function createTransaction(
         ? await apiRequest(route, options)
         : await localVet360.createTransaction();
 
-      // We want the validateAddreses method handling dataLayer events for saving / updating addresses.
-      if (vaProfileUseAddressValidation(state)) {
-        if (!fieldName.toLowerCase().includes('address')) {
-          recordEvent({
-            event:
-              method === 'DELETE' ? 'profile-deleted' : 'profile-transaction',
-            'profile-section': analyticsSectionName,
-          });
-        }
-      } else {
+      // We want the validateAddresses method handling dataLayer events for saving / updating addresses.
+      if (!fieldName.toLowerCase().includes('address')) {
```

Comment:
It would be cool if could use a `Set` containing the `fieldName`s as constants instead of the string check